conf/log4j2.properties

```diff
@@ -0,0 +1,135 @@
+#/**
```

Comment:
Why can't we continue to use the existing properties file that ships in HEAD of branch-2? It is because even with the Log4J 1 compatibility stuff we still can't make it compatible with the log4j1 properties files? 

For operational compatibility in minor releases, after users upgrade their existing logging configuration should still work. It's fine to recommend that they make changes, like a switch to the XML format, to gain access to improvements, but it would appear log4j has decided to make arbitrary changes that break existing configurations still, so logging wouldn't work if the user doesn't port. 

Here are the first few lines from log4j.properties in branch-2. 

    # Define some default values that can be overridden by system properties
    hbase.root.logger=INFO,console
    hbase.security.logger=INFO,console
    hbase.log.dir=.
    hbase.log.file=hbase.log
    hbase.log.level=INFO
    # Define the root logger to the system property "hbase.root.logger".
    log4j.rootLogger=${hbase.root.logger}
    # Logging Threshold
    log4j.threshold=ALL
    #
    # Daily Rolling File Appender
    #
    log4j.appender.DRFA=org.apache.log4j.DailyRollingFileAppender
    log4j.appender.DRFA.File=${hbase.log.dir}/${hbase.log.file}

It looks like the new properties file format drops the "log4j" prefix? Can we keep it? 

Or, we could implement logic in the launch script that looks at the user's existing log4j properties file and rewrites it, and uses the rewritten version, if it detects this?